code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -1,1053 +1,656 @@
+
+/datum/emote/living/carbon/human/scream/screech/should_play_sound(mob/user, intentional)
+	if(ismonkeybasic(user))
+		return TRUE
+	return ..()
+
+/datum/emote/living/carbon/human/gasp
+	key = "gasp"
+	key_third_person = "gasps"
+	message = "gasps!"
+	message_mime = "appears to be gasping!"
+	emote_type = EMOTE_SOUND
+	age_based = TRUE
+	volume = 100
+
+/datum/emote/living/carbon/human/gasp/get_sound(mob/user)
+
+	var/mob/living/carbon/human/H = user
+
+	if(H.is_muzzled())
+		// If you're muzzled you're not making noise
+		return
+
+	if(H.health > 0)
+		return H.dna.species.gasp_sound
+
+	if(H.gender == FEMALE)
+		return pick(H.dna.species.female_dying_gasp_sounds)
+	else
+		return pick(H.dna.species.male_dying_gasp_sounds)
+
+/datum/emote/living/carbon/human/shake
+	key = "shake"
+	key_third_person = "shakes"
+	message = "shakes their head."
+	message_param = "shakes their head at %t."
+/datum/emote/living/carbon/human/pale
+	key = "pale"
+	message = "goes pale for a second."
+
+/datum/emote/living/carbon/human/raise
+	key = "raise"
+	key_third_person = "raises"
+	message = "raises a hand."
+	hands_use_check = TRUE
+
+/datum/emote/living/carbon/human/salute
+	key = "salute"
+	key_third_person = "salutes"
+	message = "salutes."
+	message_param = "salutes to %t."
+	hands_use_check = TRUE
+
+/datum/emote/living/carbon/human/shrug
+	key = "shrug"
+	key_third_person = "shrugs"
+	message = "shrugs."
+
+/datum/emote/living/carbon/human/sniff
+	key = "sniff"
+	key_third_person = "sniff"
+	message = "sniffs."
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/living/carbon/human/johnny
+	key = "johnny"
+	key_third_person = "johnnys"  // ?????
+	message = "takes a drag from a cigarette and blows their own name out in smoke."
+	emote_type = EMOTE_AUDIBLE | EMOTE_MOUTH
+
+/datum/emote/living/carbon/human/johnny/select_param(mob/user, params)
+	if(!params)
+		return message
+	var/target = params
+	var/msg
+	for(var/mob/A in oview(5, user))
+		if(target == A.name)
+			if(user.mind?.miming)
+				msg = "takes a drag from a cigarette and blows \"[A.name]\" out in smoke."
+			else
+				msg = "says, \"[A.name], please. They had a family.\" [user] takes a drag from a cigarette and blows [user.p_their()] name out in smoke."
+			return msg
+	return message
+
+/datum/emote/living/carbon/human/johnny/run_emote(mob/user, params, type_override, intentional)
+	var/mob/living/carbon/human/H = user
+	if(!istype(H.wear_mask, /obj/item/clothing/mask/cigarette))
+		to_chat(user, "<span class='warning'>You can't be that cool without a cigarette between your lips.</span>")
+		return
+
+	if(H.getOxyLoss() > 30)
+		var/obj/item/clothing/mask/cigarette/cig = H.wear_mask
+		to_chat(user, "<span class='warning'>You gasp for air and swallow your [cig]!</span>")
+		if(cig.lit)
+			to_chat(user, "<span class='userdanger'>The lit [cig] burns on the way down!")
+			cig.forceMove(user)
+			H.wear_mask = null
+			H.update_inv_wear_mask()
+			qdel(cig)
+			H.adjustFireLoss(5)
+		return
```

Comment:
return TRUE here. otherwise you get `Unusable emote 'johnny'. Say *help for a list. ` and the emote definitelly was usable.